app/web/features/communities/events/EventCard.tsx

```diff
@@ -17,65 +17,66 @@ import { useMemo } from "react";
 import { routeToEvent } from "routes";
 import { timestamp2Date } from "utils/date";
 import dayjs from "utils/dayjs";
-import makeStyles from "utils/makeStyles";
 import stripMarkdown from "utils/stripMarkdown";
 
-const useStyles = makeStyles<Theme, { eventImageSrc: string }>((theme) => ({
-  root: {
-    "&:hover": {
-      backgroundColor: theme.palette.grey[50],
-    },
+const StyledCard = styled(Card, {
+  shouldForwardProp: (prop) => prop !== "isCancelled",
+})<{ isCancelled?: boolean }>(({ theme, isCancelled }) => ({
+  position: "relative",
+  "&:hover": {
+    backgroundColor: theme.palette.grey[50],
   },
-  cancelledChip: {
-    backgroundColor: theme.palette.error.main,
-    color: theme.palette.common.white,
-    fontWeight: "bold",
-  },
-  cancelledEvent: {
+  ...(isCancelled && {
     opacity: 0.6,
     backgroundColor: theme.palette.grey[200],
-  },
-  image: {
-    padding: theme.spacing(1),
-    backgroundColor: theme.palette.grey[200],
-    height: 80,
-    backgroundImage: ({ eventImageSrc }) => `url(${eventImageSrc})`,
-    backgroundSize: ({ eventImageSrc }) =>
-      eventImageSrc === eventImagePlaceholderUrl ? "contain" : "cover",
-    [theme.breakpoints.up("sm")]: {
-      height: 100,
-    },
-    [theme.breakpoints.up("md")]: {
-      height: 120,
-    },
-  },
-  chip: {
-    borderRadius: theme.shape.borderRadius,
-    fontWeight: "bold",
-  },
-  title: {
-    display: "-webkit-box",
-    boxOrient: "vertical",
-    lineClamp: 2,
-    overflow: "hidden",
-  },
-  eventTime: {
-    display: "-webkit-box",
-    boxOrient: "vertical",
-    lineClamp: 2,
-    overflow: "hidden",
-    [theme.breakpoints.up("sm")]: {
-      lineClamp: 1,
-    },
-  },
-  content: {
-    display: "-webkit-box",
-    boxOrient: "vertical",
-    lineClamp: 5,
-    overflow: "hidden",
+  }),
+}));
+
+const CardWrapper = styled("div", {
+  shouldForwardProp: (prop) => prop !== "isCancelled",
+})({
+  position: "relative",
+});
```

Comment:
You are not passing `isCancelled` in the props to this element, not using it, so I think you can remove the `isCancelled` logic here. I'm also unsure if you need this element? We didn't have an additional wrapper before.

You can probably set the `CardMedia` element itself to `position: absolute` and/or give it negative margins.